UI/Services/DistributedSessionService.cs

```diff
@@ -1,12 +1,16 @@
 using System.Text;
 using Application.Common.Interfaces;
 using Newtonsoft.Json;
+using Polly;
+using IntegerConstants = UI.Constants.IntegerConstants;
 
 namespace UI.Services;
 
 public class DistributedSessionService : ISessionService
 {
     private const string DefaultPreKey = "General";
+    public const string FormPostPreKey = "FormPostPreKey";
```

Comment:
make it `public` as the other one

Reply:
Assuming mean `private`?  I have done that.